vpn/src/main/java/com/duckduckgo/mobile/android/vpn/pixels/DeviceShieldPixels.kt

```diff
@@ -587,6 +598,22 @@ class RealDeviceShieldPixels @Inject constructor(
         firePixel(DeviceShieldPixelNames.ATP_DID_CHOOSE_CANCEL_APP_PROTECTION_DIALOG)
     }
 
+    override fun didShowVpnConflictDialog() {
+        firePixel(DeviceShieldPixelNames.ATP_DID_SHOW_VPN_CONFLICT_DIALOG)
+    }
+
+    override fun didChooseToDismissVpnConflicDialog() {
+        firePixel(DeviceShieldPixelNames.ATP_DID_CHOOSE_DISMISS_VPN_CONFLICT_DIALOG)
+    }
+
+    override fun didChooseToOpenSettingsFromVpnConflicDialog() {
+        firePixel(DeviceShieldPixelNames.ATP_DID_CHOOSE_OPEN_SETTINGS_VPN_CONFLICT_DIALOG)
+    }
+
+    override fun didChooseToContinueFromVpnConflicDialog() {
+        firePixel(DeviceShieldPixelNames.ATP_DID_CHOOSE_OPEN_SETTINGS_VPN_CONFLICT_DIALOG)
+    }
```

Comment:
Would we also want daily pixels to see how many users hit this issue? I think that metric would be more interesting or at least complementary to the absolute count